packages/drivers/driver-base/src/documentDeltaConnection.ts

```diff
@@ -156,6 +158,26 @@ export class DocumentDeltaConnection
 				0x20b /* "mismatch" */,
 			);
 			if (!this.trackedListeners.has(event)) {
+				if (event === "pong" && this.trackLatencyTimer === undefined) {
+					// log latency every minute
+					this.trackLatencyTimer = setInterval(() => {
+						const start = Date.now();
+
+						// emit pong event every minute. If latency is longer than 1 min, log separately
+						this.socket.emit("pong", () => {
```

Comment:
Wondering if this might be a good use case for [volatile events](https://socket.io/docs/v4/emitting-events/#volatile-events). We probably don't care to buffer the pings if the connection to the server is down?